src/main/java/com/ericsson/ei/subscription/RunSubscription.java

```diff
@@ -52,6 +55,8 @@
     @Getter
     @Value("${spring.data.mongodb.database}")
     public String dataBaseName;
+    
+    Map<String, List<String>> subscriptionsCache = new HashedMap<>();
```

Comment:
We can move this into the SubscriptionRepeatDBHandler and consider using https://docs.spring.io/spring-framework/docs/current/reference/html/integration.html#cache-store-configuration-caffeine